### ISSUE TITLE
bodhi should re-run auto QA tests if a build is edited and packaged changed

### ISSUE THREAD

**Issue by nullr0ute** [fedora-infra/bodhi#846]
In this update the grub2 NVR is changed from grub2-2.02-0.33.fc24 to grub2-2.02-0.34.fc24 yet the test results are against -0.33. I would expect if a build is changed that the tests would be re-run to ensure no regressions against the new NVR. I would expect the old test results to remain for reference and the new set of test results to also be displayed.

See example where the tests are reported:
https://bodhi.fedoraproject.org/updates/FEDORA-2016-c4d43baacc

**Comment by lmacken** [comment]
Yeah, something weird happened with this update. Bodhi allowed multiple builds for the same update in there, which I had to remove manually today. Needs further investigation & a unit test to reproduce this issue.
